examples/lines_bars_and_markers/linestyles.py

```diff
@@ -3,50 +3,65 @@
 Linestyles
 ==========
 
-This examples showcases different linestyles copying those of Tikz/PGF.
+Linestyle can be provided as simple as *solid* , *dotted*, *dashed*
+or *dashdot*. Moreover, the dashing of the line can be controlled by
+a dash tuple such as (offset, (on_off_seq)) as mentioned in
+`.Line2D.set_linestyle`. For e.g., ``(0, (3, 10, 1, 15))`` means
+ 3pt-line,10pt-space,1pt-line,15pt-space with no offset.
+
+*Note*: The dash style can also be configured via `.Line2D.set_dashes`
+as shown in :doc:`/gallery/lines_bars_and_markers/line_demo_dash_control`
+and passing a list of dash sequences using the keyword *dashes* to the
+cycler in :doc:`property_cycle </tutorials/intermediate/color_cycle>`.
 """
 import numpy as np
 import matplotlib.pyplot as plt
-from collections import OrderedDict
 from matplotlib.transforms import blended_transform_factory
 
-linestyles = OrderedDict(
-    [('solid',               (0, ())),
-     ('loosely dotted',      (0, (1, 10))),
-     ('dotted',              (0, (1, 5))),
-     ('densely dotted',      (0, (1, 1))),
+linestyle_str = [
+     ('solid',                 ('solid')),    # Same as (0, ()) or '-'
```

Comment:
The parentheses around the second element do nothing; are they necessary for something?